about.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <title>Naledi's Capstone</title>
+  </head>
+
+  <body>
+    <header>
+      <ul
+        class="hidden items-center bg-gray-800 text-white md:flex justify-end h-[30px] pr-[10%]"
+      >
+        <li class="p-1">
+          <a href=""><i class="bi bi-facebook"></i></a>
+        </li>
+        <li class="p-1">
+          <a href=""><i class="bi bi-twitter"></i></a>
+        </li>
+        <li class="p-1"><a href="">English </a></li>
+        <li class="p-1"><a href="">My Page</a></li>
+        <li class="p-1"><a href="">logout</a></li>
+      </ul>
+
+      <!-- navigation -->
+
+      <nav class="px-[10%] md:flex h-[55px] justify-between shadow-xl">
+        <i class="menulist bi bi-list md:hidden p-4 text-[40px] pl-0"></i>
+        <a class="p-3 md:flex" href="index.html"
+          ><img src="assets/wine-glass.png" alt="" />wine~up</a
+        >
+        <ul class="items md:flex items-center justify-center">
+          <li class="close"><i class="bi bi-x-lg"></i></li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="index.html">Home</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="">Program</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]"><a href="">Join</a></li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]">
+            <a href="">Sponsor</a>
+          </li>
+          <li class="item p-3 m-1 hover:text-[#ec5242]"><a href="">News</a></li>
+          <li
+            class="item p-1 m-1 hover:text-[#ec5242] border-solid border-4 border-[#ec5242]"
+          >
+            <a href="">CC Campaign</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
```

Comment:
The mobile navbar is not working. Kindly fix this issue. I have recorded [this video](https://www.loom.com/share/995ea0b1c5eb4346a5f0d096bb8e3b5b) where you can verify the problem.